apps/src/lib/ui/VisualizationResizeBar.jsx

```diff
@@ -110,14 +110,16 @@ class VisualizationResizeBar extends React.Component {
         window.innerWidth -
         (window.pageXOffset + rect.left + rect.width / 2) -
         parseInt(window.getComputedStyle(this.domElement).right, 10);
-      newVizWidth = window.innerWidth - event.pageX - offset;
+      const pageX = event.pageX || (event.touches && event.touches[0].pageX);
```

Comment:
Do you know if this will have the same problem with deduplication as with the vertical resizer?

Reply:
It works fine on Chrome on a Samsung SM-T380 Android tablet, and I believe also on your Android phone.  Interestingly, I feel like it renders faster than the vertical resizer, though I don't know why that would be.